apps/bare-expo/ios/Podfile.lock

```diff
@@ -456,17 +456,17 @@ PODS:
     - React
   - react-native-netinfo (5.9.2):
     - React
-  - react-native-safe-area-context (3.0.2):
+  - react-native-safe-area-context (3.1.1):
     - React
-  - react-native-segmented-control (1.6.1):
+  - react-native-segmented-control (2.1.0):
     - React
-  - react-native-slider (3.0.0):
+  - react-native-slider (3.0.3):
```

Comment:
These were also updated. I guess this is intentional?

Reply:
Yes, they weren't updated while I was upgrading those (upgrades already landed on `master`), so I made this PR as "revert problem" and "fix-all". 👍